draft-duke-quic-load-balancers.md

```diff
@@ -59,270 +61,545 @@ author:
+
+   These message formats are specific to QUICv2 and experimental
+   versions leading up to QUICv2. They may require revision for
+   future versions of QUIC.
+
+## Packet Header Format
+   QUIC-LB messages resemble QUIC packets with a long header. They are
+   sent when a load balancer boots up, or detects a new server in the
+   pool. QUIC-LB packets are delivered in a UDP datagram.
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+
+| Type = 0xfb   |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                      	Version (32)                         |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|      0x00     |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                           Length (i)                          |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|     Random    |
++-+-+-+-+-+-+-+-+
+{: #quic-lb-header title="QUIC-LB Header"}
+
+   The type field is 0xfb, which is otherwise unused in QUICv2.
+
+   The Version field allows QUIC-LB to use the Version Negotiation
+   mechanism. All messages in this specification are specific to
+   QUICv2, as future QUIC versions may use the 0xfb type for other
+   purposes. Therefore, the Version field should be set as 0xff00000d,
+   the current QUIC version.
+
+   Load balancers MUST cease sending QUIC-LB packets of this version
+   to a server when that server sends a Version Negotiation packet
+   that does not advertise the version.
+
+   The 0x00 byte indicates that there are no connection IDs present
+   in the header.
+
+   The length field is a variable-length integer that encodes the
+   length of the entire QUIC-LB packet, in octets, after the length
+   field. The variable integer encoding is the same as that in
+   {{QUIC-TRANSPORT}}.
+   
+   The Random field is one random octet, meant to emulate an
+   encrypted one-octet QUIC packet number. 
+
+   The remainder of the packet is the payload. This has multiple
+   formats.
+
+### Ack Payload
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|  Type = 0x00  |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                            Token (64)                         +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+{: #ack-payload title="Ack Payload"}
+   The Ack Payload consists of a nine octets. Servers send this
+   payload after receipt of any acceptable QUIC-LB packet from a load
+   balancer.
+
+   The token field echoes the token field from the acknowledged
+   packet.
```

Comment:
I see no reason to have a fixed size for `Token` or at least not one so large. I'm on the fence about using it all together.

Reply:
It seems easier to me, and the cost of using a big token is not high. But if you suggest alternate text I'm don't feel strongly about it.